pkg/apis/core/types.go

```diff
@@ -2686,6 +2686,12 @@ type PodSpec struct {
 	// If not specified, the default is true.
 	// +optional
 	EnableServiceLinks *bool
+	// TopologySpreadConstraints describes how a group of pods ought to spread across topology
+	// domains. Scheduler respects and enforces the constraints.
```

Comment:
nit: `respects this during scheduling`, enforcing sounds like it does something at runtime to enforce it or some other wording ?